pyvista/plotting/volume_property.py

```diff
@@ -231,6 +231,32 @@ def shade(self) -> bool:
     def shade(self, value: bool):
         self.SetShade(value)
 
+    @property
+    def independent_components(self) -> bool:
+        """Return or set independent components.
+
+        If ``False``, then you must have either 2 or 4 component data.
+        For 2 component data, the first is passed through the
+        first color transfer function and the second component is passed
+        through the first scalar opacity (and gradient opacity) transfer
+        function. Normals will be generated off of the second component. When
+        using gradient based opacity modulation, the gradients are computed off
+        of the second component.
+
+        For 4 component data, the first three will directly represent RGB (no
+        lookup table). The fourth component will be passed through the first
+        scalar opacity transfer function for opacity and first gradient opacity
+        transfer function for gradient based opacity modulation. Normals will
```

Comment:
This needs to be clearly stated in `add_volume()` as well. The `opacity` argument still has a strong affect on the visualization of RGBA volumes which IMO is a bit unexpected.